src/internal/operators/share.ts

```diff
@@ -224,15 +231,15 @@ export function share<T>(options: ShareConfig<T> = {}): MonoTypeOperatorFunction
             dest.complete();
           },
         });
-        from(source).subscribe(connection);
+        source.subscribe(connection);
```

Comment:
i dont know why a `from` was here, since it is an observable already, but if at all we should probably use `innerFrom`?